### PR TITLE
Adds new reagent application method, INJECT.  This will hopefully pre…

### LINKED ISSUE #9305: Some reagents add 1.5x of their amount when applied by patch.
patch

I don't have the time to test this with every single reagent, but results seem to be consistent with each. Some reagents, such as silver sulf (As pictured) give 50% more when applied with a patch. Other reagents, such as water, welding fuel, and sulfur give the proper amount. And then there's styptic powder which gives its amount when self-applied, but has the 1.5x boost when applied to someone else? How the fuck?

### PR DESCRIPTION
**Pull request by lordpidey**
Adds new reagent application method, INJECT.  This will hopefully prevent people from thinking they shouldn't have eaten Styptic powder after being injected with it.

Also, as a side effect, fixes #9305 for itching powder, but if the problem still exists for other chemicals, then it's still around.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm [hunk 1]
```diff
@@ -627,6 +627,9 @@
 	if(method == INGEST)
 		C.take_organ_damage(min(6*toxpwr, reac_volume * toxpwr))
 		return
+	if(method == INJECT)
+		C.take_organ_damage(1.5 * min(6*toxpwr, reac_volume * toxpwr))
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm]: why is this doing the retarded take_organ_damage proc again

can we get rid of that shit and replace it with AdjustBruteLoss like sane people

**Comment by Iamgoofball** [comment]
:+1:

shoulda added this ages ago

**Comment by Shapsy** [comment]
Just change the styptic/sulf/synthflesh message to say you shouldn't ingest it

**Comment by Iamgoofball** [comment]
That's what the message said before, but someone changed it because muh immuhlsions

**Pull request merged**